docs/src/snap/howto/install/install-offline.md

```diff
@@ -0,0 +1,374 @@
+# Installing Canonical Kubernetes Offline or in an air gapped environment
+
+There are situations where it is necessary or desirable to run Canonical 
+Kubernetes on a machine that is not connected to the internet. 
+Based on different degrees of separation from the network,
+different solutions are offered to accomplish this goal.
+This guide explains the necessary preparation required for the
+offline installation and walks you through the different potential scenarios.
+
+## Prepare for Deployment
+
+In preparation for the offline deployment you will download the Canonical
+Kubernetes snap, fulfill the networking requirements based on your scenario and
+handle images for workloads and Canonical Kubernetes features.
+
+### Prep 1: Download the Canonical Kubernetes snap
+
+From a machine with access to the internet download the following:
+
+```bash
+sudo snap download k8s --channel 1.30-classic/beta
+sudo snap download core20
+sudo mv k8s_*.snap k8s.snap
+sudo mv k8s_*.assert k8s.assert
+sudo mv core20_*.snap core20.snap
+sudo mv core20_*.assert core20.assert
+```
+
+The [core20][Core20] and `k8s` snap are downloaded. The `core20.assert` and 
+`k8s.assert` files, are necessary to verify the integrity of the snap packages.
+
+```{note} Update the version of k8s by adjusting the channel parameter.
+    Find the version you desire in the [snapstore](https://snapcraft.io/k8s).```
+
+```{note} With updates to the snap the base core is subject to change.```
+
+### Prep 2: Network Requirements
+
+Air-gap deployments are typically associated with a number of constraints and
+restrictions with the networking connectivity of the machines.
+Verify that your cluster nodes can communicate, machines have a default gateway
+and optionally ensure proxy access.
+
+#### Network Requirement: Cluster node communication
+<!-- TODO: Services and Ports Doc -->
+Ensure that all cluster nodes are reachable from each other.
+Refer to [Services and ports][svc-ports] used for a list of all network ports
+used by Canonical Kubernetes.
+
+#### Network Requirement: Default Gateway
+
+Kubernetes services use the default network interface of the machine
+for the means of node discovery:
+
+- kube-apiserver (part of kubelite) 
+  - uses the default network interface to advertise this address to 
```

Comment:
this was the case for microk8s. in canonical kubernetes, the node address is defined and well-known when bootstrapping the cluster. instead of writing this, let's instead include the `--advertise-address` flag in kube-apiserver (using the node IP)

Reply:
this would render the whole section about a default gateway irrelevant, as we can fully configure the necessary services